database/migrations/2021_11_18_182110_create_reservations_table.php

```diff
@@ -18,7 +18,7 @@ public function up(): void
             $table->dateTime('starts_at');
             $table->dateTime('ends_at');
             $table->foreignIdFor(Space::class);
-            $table->foreignIdFor(User::class, 'created_by_user_id');
+            $table->foreignIdFor(AuthorizedUser::class, 'created_by_user_id');
```

Comment:
Misschien handiger om dan ook de column naam aan te passen? Voor de context binnen de database is dit nog handig.

```suggestion
            $table->foreignIdFor(AuthorizedUser::class, 'created_by_authorized_userr_id');
````